TIPs/TIP-181.md

```diff
@@ -0,0 +1,65 @@
+| id    | Title | Status      | Author  | Description | Discussions to | Created    |
+| ----- | ----- | ----------- | ------- | ----------- | -------------- | ---------- |
+| TIP-181 | Chainlink CCIP THALES Cross-chain Staking | Draft | kirila | Use CCIP to unify THALES staking across all supported networks | [discordlink ](https://discord.gg/thales)   | 2023-11-08 |
+
+
+## Simple Summary
+
+
+Use Chainlink Cross-Chain Interoperability Protocol (CCIP) to synchronize THALES token staking across all supported networks.
+
+
+## Abstract
+
+Currently THALES staking is separately deployed on Optimism, Arbitrum and Base with their own isolated reward pools and isolated rewards calculations.
+
+
+This TIP proposes synchronizing staking rewards across all supported networks by unifying all the dedicated-per-chain rewards into a concept of total-cross-chain rewards using Chainlink Cross-Chain Interoperability Protocol.  
+ 
+At the end of every staking period, the Chainlink Cross-Chain Interoperability Protocol (CCIP) is used to distribute the total rewards across all chains.
+
+
+## Motivation
+
+Currently, the staking rewards on every supported network (Optimism, Arbitrum, and Base) are distributed independently from each other with isolated rewards pools.
+This causes each network to have different base and  bonus staking rewards each week from other staking instances on other networks.
+Stakers on one network may receive less rewards for the equal amount of staked THALES and less gamified staking points compared to other stakers on other networks. This additionally had the causation of some stakers aiming to unstake from one network, bridge and stake to another network to chase better yield. With this proposal, this action will no longer be necessary.
```

Comment:
Think this would be better:
"Additionally, this led to some stakers attempting to unstake from one network, bridge their assets, and stake them on another network in pursuit of better yield. With this proposal, such actions will no longer be necessary."